parsec/core/gui/forms/greet_user_check_info_widget.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>596</width>
-    <height>356</height>
+    <width>413</width>
+    <height>496</height>
```

Comment:
Is that on purpose?

Reply:
Yes and no, it doesn't really matter and there's nothing I can do about it. QDesigner does not give you the option to hide widgets, so when you add a new one like a spinner, your basic widget will take more space. It's not important because we do hide some widgets in the code, and the real size will be calculated by the layouts.